### PR TITLE
CLUS-firepower-restapi110-md1

### PR DESCRIPTION
**Pull request by ReneeBornstein**
* changed sentence strutures
* moved links to earlier parts of
  sentence to avoid periods
* lengthened sentence spacing

### CODE REVIEW

File: labs/firepower-restapi-110/1.md [hunk 1]
```diff
@@ -1,80 +1,71 @@
 # FMC - REST - API - 110
 
-## Lab Objective
+## Lab objective
```
Comment (paulzim) [inline labs/firepower-restapi-110/1.md]: Should just be "Objectives"

**Review by paulzim — CHANGES_REQUESTED** [review]
Just a minor fix

File: labs/firepower-restapi-110/1.md [hunk 2]
```diff
@@ -1,80 +1,71 @@
...
 
-Expose developers to using the Event Streamer (eStreamer) API to extract events
-from the Firepower Management Center (FMC) and perform basic analysis of a
-potentially compromised host.
 
 ## Prerequisites
-Basic understanding of Firepower Solutions and Firepower threat technologies
+It's best if you already have basic understanding of Firepower Solutions and Firepower threat technologies.
+
+## Learning objective
```
Comment (annegentle) [inline labs/firepower-restapi-110/1.md]: Can eliminate the "Learning objective" heading here.

File: labs/firepower-restapi-110/1.md [hunk 3]
```diff
@@ -1,80 +1,71 @@
...
-    on the eStreamer server and creates an authentication certificate to use
-    during client-server authentication.
+  * If you want to encrypt the certificate file, enter a password in the **Password** field.
+
+  * Click **Save**. The eStreamer server now allows the host to access port 8302 on the eStreamer server, and creates an    authentication certificate to use during client-server authentication.
 
-  * Click the download file icon () next to the client hostname to download the
-    certificate file and copy it to the system on which you plan to run the
-    eStreamer client.
+  * Click the download file icon () next to the client hostname to download the certificate file. Copy it to the system on which you plan to run the eStreamer client.
```
Comment (annegentle) [inline labs/firepower-restapi-110/1.md:45]: There are a couple of empty parens `()` here and below, should those simply be deleted?

File: labs/firepower-restapi-110/1.md [hunk 4]
```diff
@@ -1,80 +1,71 @@
...
 
-  * To revoke access for a client, click the delete icon () next to the host you
-    want to remove. Note that you do not need to restart the eStreamer service;
-    access is revoked immediately.
+  * To revoke access for a client, click the delete icon () next to the host you want to remove. Note that you do not need to restart the eStreamer service; access is revoked immediately.
 
     `Note: In a multi domain deployment, you can create an eStreamer client in any domain. The authentication certificate allows the client to request events only from the client certificate's domain and any descendant domains. The eStreamer configuration page shows only clients associated with the current domain, so if you want to download or revoke a certificate, switch to the domain where the client was created.`
 
-* Download the version of the eStreamer SDK which matches the version of your FMC from <https://developer.cisco.com/site/firepower/docs/downloads/estreamer-api/>
+* From <https://developer.cisco.com/site/firepower/docs/downloads/estreamer-api/> download the version of the eStreamer SDK which matches the version of your FMC.
```
Comment (annegentle) [inline labs/firepower-restapi-110/1.md:51]: Use Markdown for the link.

**Review by annegentle — CHANGES_REQUESTED** [review]
Some comments inline.

**Comment by ReneeBornstein** [comment]
This one same as last email you sent:

OMG, another weird error. I opened the GitHub doc link you sent, clicked on Files changed, and copied that to another window, so I could edit it. Usually I then just click on the pencil icon to edit the file. But this time I got this error:

[/var/folders/9x/dk6rpbtd6tnc0tqvtk73lz100000gn/T/com.microsoft.Outlook/WebArchiveCopyPasteTempFiles/cidimage001.png@01D40BD1.FE73A850]

I’ve never had that happen before. I am logged in, so I guess somehow my push access (whatever that is) suddenly isn’t working?

**Review by annegentle — APPROVED** [review]

**Pull request merged**